### DISCUSSION TITLE
nextjs 12 breaks import bootstrap as scss

### DISCUSSION THREAD

**Discussion by teeto** [vercel/next.js#31141] [Help]
I have created a new project using new version nextjs v12
I want to use bootstrap and i am doing as i do in v11.
First i install last versión of bootstrap:
npm install bootstrap@latest

It installs bootstrap 5.1.3

Then in _app.js i write this line:
import "styles/custom.scss";

And then in custom.scss i write:
@import "node_modules/bootstrap/scss/bootstrap.scss";

As it is, it works in v11, but in v12 i get this error:

`Module not found: Can't resolve '../node_modules/bootstrap/scss/forms/data:image/svg+xml,<svg xmlns='http:/www.w3.org/2000/svg' viewBox='0 0 16 16'><path fill='none' stroke=''`

**Comment by Dragate** [comment]
Having the same issue on 12.0.3 specifically. It does not occur on 12.0.0-12.0.2.

My `.babelrc` looks like this:
```json
{
  "presets": [
    [
      "next/babel",
      {
        "styled-jsx": {
          "plugins": [
            [
              "styled-jsx-plugin-sass",
              {
                "sassOptions": {
                  "includePaths": [
                    "./site/styles",
                    "./node_modules"
                  ],
                  "precision": 2,
                  "data": "@import 'base';"
                }
              }
            ]
          ]
        }
      }
    ]
  ]
}
```

**Comment by teeto** [comment]
Thanks, you are right, it seems a 12.0.3 bug, i have downgraded to 12.0.2 and it works

**Comment by zbialecki** [comment]
I'm also experiencing this issue. I think it's a bug with the version of Webpack that ships with 12.0.3. Importing any scss file that uses data urls throws an error `HookWebpackError: Cannot find module './data:image/svg+xml;base64,...`

**Comment by teeto** [comment]
It seems fixed in last version 12.0.7
